c7n/resources/s3.py

```diff
@@ -1495,40 +1491,16 @@ def process(self, buckets):
 
     def process_bucket(self, bucket):
         s3 = bucket_client(local_session(self.manager.session_factory), bucket)
-        state = self.data.get('state', 'enable')
-        scopes = self.data.get('scope', ['All'])
         try:
-            config = s3.get_public_access_block(
-                Bucket=bucket['Name'])['PublicAccessBlockConfiguration']
-        except ClientError as e:
-            if e.response['Error']['Code'] == 'NoSuchPublicAccessBlockConfiguration':
-                # Set config to none because NoSuchPublicAccessBlockConfiguration error
-                # was returned. This is important later because it is symbolic of
-                # all blocks being set to false. See if/else statement below
-                config = None
-            else:
-                raise
-        if config:
-            if 'All' in scopes:
-                for key in config.keys():
-                    config[key] = True if state == 'enable' else False
-            else:
-                for scope in scopes:
-                    config[scope] = True if state == 'enable' else False
+            config = dict()
```

Comment:
use config = {} .. basic type class as constructors aren't pythonic

Reply:
Awesome! I'm a little bit newer to python so this kind of feedback is really helpful. I am noticing I have some bad habits with me from Java ;). Resolved in 2293123

Reply:
if in doubt on the zen of python...

```
$ python3
>>> import this
```
for a koan from Tim Peters

Reply:
This is really cool! I think I might want to print a screenshot and place somewhere around the home office haha. Explicit over implicit, this one has been getting me quite a bit. Was thinking around the context of a rules engine/config management tool, this is also a zen to live by.